plaso/storage/fake/writer.py

```diff
@@ -524,6 +502,27 @@ def WriteSessionCompletion(self, aborted=False):
     self._session.aborted = aborted
     self.session_completion = self._session.CreateSessionCompletion()
 
+  # pylint: disable=unused-argument
+  def WriteSessionConfiguration(self, knowledge_base):
+    """Writes session configuration information.
+
+    Args:
+      knowledge_base (KnowledgeBase): contains information about the session
+          configuration.
+
+    Raises:
+      IOError: if the storage type does not support writing preprocessing
+          information or when the storage writer is closed.
+      OSError: if the storage type does not support writing preprocessing
+          information or when the storage writer is closed.
+    """
+    self._RaiseIfNotWritable()
+
+    if self._storage_type != definitions.STORAGE_TYPE_SESSION:
+      raise IOError('Session configuration not supported by storage type.')
```

Comment:
Please add the actual storage type to the error message here

Reply:
Same reply as previous comment.